features/step_definitions/gxt_table_steps.rb

```diff
@@ -20,11 +20,12 @@ def self.accessor_methods(accessor, name)
         self.send("#{name}_element").rows
       end
     end
-    
+
     protected
-      def child_xpath
-        ".//descendant::tr"
-      end
+
+    def strategy
+      :descendants
+    end
```

Comment:
I'm still digging through PO code/documentation. I realized that this code breaks the things in the widget documentation. This looks like pretty single use case specific instance for gwt/gxt (which I have no idea what it is), and it appears that the only actual user feature for it is "descendants" vs direct "children". If those are the only two strategies, the implementation with the latest adjacent code in Watir is [really easy without needing the user to specify xpath](https://github.com/cheezy/page-object/pull/436/files#diff-fc06631e0c4056ad3b636c74c00908b2R49)

I can make this code backward compatible by putting `#child_xpath` back and parsing it to work for the new code. Can also create a deprecation notice for it. Let me know what you think